### PR TITLE
Adding new authorisation providers: dev environment 

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- testing experiments with google, huawei and gitee
- dev configuration for oauth providers (no prod keys)
- frontend changes to support it

Mouse is over Github on the page:

image

### CODE REVIEW

**Review by sanyavertolet — APPROVED** [review]
lgtm but Huawei logo needs to be aligned properly

**Comment by akuleshov7** [comment]
> lgtm but Huawei logo needs to be aligned properly as well as Gitee one

Easy to say, but yes, it’s my focus )

**Comment by sanyavertolet** [comment]
> > lgtm but Huawei logo needs to be aligned properly as well as Gitee one
> 
> Easy to say, but yes, it’s my focus )

At least you could try `align-middle` on `div` that contains the logo. It will not fix sizes, but still

File: api-gateway/src/main/resources/application-dev.yml [hunk 1]
```diff
@@ -15,11 +15,16 @@ spring:
...
+          huawei:
+            authorization-uri: https://oauth-login.cloud.huawei.com/oauth2/v3/authorize
+            token-uri: https://oauth-login.cloud.huawei.com/oauth2/v3/token
+            user-info-uri: https://oauth-login.cloud.huawei.com/oauth2/v3/userinfo
+            user-name-attribute: name
           gitee:
             authorization-uri: https://gitee.com/oauth/authorize
             token-uri: https://gitee.com/oauth/token
             user-info-uri: https://gitee.com/api/v5/user
+          # example: https://github.com/wearearima/spring-boot-dex/blob/master/src/main/resources/application.properties
```
Comment (kgevorkyan) [inline api-gateway/src/main/resources/application-dev.yml:27]: if its necessary info, probably better to link the concrete commit hash, i.e permalink

```suggestion
          # example: https://github.com/wearearima/spring-boot-dex/blob/ad7dd7f501f8fc3946fc87184b4b8b705713ead9/src/main/resources/application.properties
```

File: api-gateway/src/main/resources/application-dev.yml [hunk 2]
```diff
@@ -52,6 +67,10 @@ spring:
             scope:
               - openid
           github:
+            # IMPORTANT! When you create a github application on the GITHUB you need to provide a callback API.
+            # If you will only provide a domain name: 'saveourtool.com' in CALLBACK URL field, everything will go well.
+            # Github will automatically
```
Comment (kgevorkyan) [inline api-gateway/src/main/resources/application-dev.yml:72]: automatically ... ?

**Review by kgevorkyan — APPROVED** [review]

**Comment by akuleshov7** [comment]
> > > lgtm but Huawei logo needs to be aligned properly as well as Gitee one
> > 
> > 
> > Easy to say, but yes, it’s my focus )
> 
> At least you could try `align-middle` on `div` that contains the logo. It will not fix sizes, but still

I have changed everything, removed font awesome (as it is prohibited)

**Pull request merged**